pages/requests/new/index.js

```diff
@@ -63,19 +69,32 @@ const NewBlankRequest = () => {
   }
 
   const handleSubmit = (event) => {
```

Comment:
```suggestion
  const handleSubmit = (event) => {
    event.preventDefault()
    event.stopPropagation()
    setValidated(true)

    if (event.currentTarget.checkValidity()) {
      if (requestForm.billingSameAsShipping === true) {
        Object.assign(requestForm.billing, requestForm.shipping)
      }
      // TODO(alishaevn): comment this back in when it's working
      // createRequest(requestForm)
      // TODO(summercook) remove this when createRequest works.
      // only console log valid requests.
      console.log(requestForm)
    }
  }
```

Reply:
it won't let me add the suggestion to the entire method since there are deleted lines. 🤷🏾‍♀️ I'm suggesting to replace the existing method with what I have above though.

looks like the event chained methods can be called whether the form is valid or not. calling them even on valid forms allows us to handle the redirect as well.

Reply:
ok, will try this

Reply:
done!